### PR TITLE
update readme about Mistral

### PR DESCRIPTION
**Pull request by EdwardLi-coder**
Fix: Readme uses outdated mistral lib #268

### CODE REVIEW

**Comment by shcheklein** [comment]
Thanks @EdwardLi-coder !

File: README.rst [hunk 1]
```diff
@@ -174,18 +174,17 @@ DataChain can parallelize API calls; the free Mistral tier supports up to 4 requ
 
 .. code:: py
```
Comment (dmpetrov) [inline README.rst:175]: Please replace `pip install mistralai` to something like `pip install mistralai>=1.0.0` (please double check the exact command).

File: README.rst [hunk 2]
```diff
@@ -174,18 +174,17 @@ DataChain can parallelize API calls; the free Mistral tier supports up to 4 requ
...
 
-    from mistralai.client import MistralClient
-    from mistralai.models.chat_completion import ChatMessage
+    from mistralai import Mistral
```
Comment (dmpetrov) [inline README.rst:177]: the code looks nicer 😻

**Review by dmpetrov — APPROVED** [review]
Looks good! Thank you for quick fix!

Only one small comment about pip install

File: README.rst (hunk 1 above)
Comment (EdwardLi-coder) [reply README.rst:175]: > Please replace `pip install mistralai` to something like `pip install mistralai>=1.0.0` (please double check the exact command).

Ok,I will update it.

**Comment by dmpetrov** [comment]
Merged. Thank you @EdwardLi-coder !!!

**Pull request merged**